Entities/Product.java

```diff
@@ -0,0 +1,67 @@
+package Entities;
```

Comment:
It is not a java convention to use `Pascal Case` naming convention for folders. They should be `Lower Case` instead